test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup_teardown/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+# tasks file for ec2
+
+- name: Create IAM role
+  iam:
+    iam_type: role
+    name: "{{iam_role_name}}"
+    state: present
+  register: role_output
+  tags: setup_infra
+
+- name: Assign a policy called SSM and S3 to the "{{iam_role_name}}"
+  iam_policy:
+    iam_type: role
+    iam_name: "{{iam_role_name}}"
+    policy_name: "{{iam_policy_name}}"
+    policy_document:  aws_ssm_integration_test_setup_teardown/files/custom_role.json
+    state: present
+  tags: setup_infra
+
+- pause:
+    seconds: 10
+  tags: setup_infra
```

Comment:
Why we need wait here?

Reply:
We are using us-east-2 region currently and after creating role, ansible was not able to pickup new role created. This is not required when change region to us-east-1.